include/arch/arc/v2/linker.ld

```diff
@@ -76,7 +82,7 @@ MEMORY {
 SECTIONS {
 
 #include <linker/rel-sections.ld>
-
+#include "linker/../../modules/embarc_mli/embarc_mli.ld"
```

Comment:
Shouldn't there be some #if here to avoid the dependency on the mli module if not used?

Reply:
Added #if to judge different conditions.